### ISSUE TITLE
[Accessibility] Unable to navigate to legend using keyboard if height not set - chart goes blank

### ISSUE THREAD

**Issue by faisaltaher1** [highcharts/highcharts#10564]
#### Expected behaviour
Using the accessibility module, users should be able to navigate to the legend using the keyboard, without explicitly setting a chart height.

#### Actual behaviour
Using the accessibility module, users are not able to navigate to the legend using the keyboard, without explicitly setting a chart height. The chart ends up going blank, or looking buggy.

#### Live demo with steps to reproduce
Example chart using existing demo:
https://jsfiddle.net/3fsy67c1/

1. Using the keyboard, attempt to tab to the legend
2. Once you expect to tab to the legend, the chart goes blank
3. If you uncomment the height property in the CSS and run steps 1 and 2, it all works fine.

#### Product version
Highcharts v7.1.1 (using accessibility module)

#### Affected browser(s)
Chrome on osx

**Comment by sebastianbochan** [comment]
Hi @faisaltaher1,
Thank you for reporting.

FYI @oysteinmoseng

**Comment by raf18seb** [comment]
+1
